src/test/regress/expected/distributed_types_conflict.out

```diff
@@ -0,0 +1,80 @@
+SET citus.next_shard_id TO 20020000;
+CREATE SCHEMA type_conflict;
+SELECT run_command_on_workers($$CREATE SCHEMA type_conflict;$$);
+       run_command_on_workers        
+-------------------------------------
+ (localhost,57637,t,"CREATE SCHEMA")
+ (localhost,57638,t,"CREATE SCHEMA")
+(2 rows)
+
+-- create a type on a worker that should not cause data loss once overwritten with a type
+-- from the coordinator
+\c - - - :worker_1_port
+SET citus.enable_ddl_propagation TO off;
+SET search_path TO type_conflict;
+CREATE TYPE my_precious_type AS (secret text, should bool);
+CREATE TABLE local_table (a int, b my_precious_type);
+INSERT INTO local_table VALUES (42, ('always bring a towel', true)::my_precious_type);
+\c - - - :master_port
+SET search_path TO type_conflict;
+-- overwrite the type on the worker from the coordinator. The type should be over written
+-- but the data should not have been destroyed
+CREATE TYPE my_precious_type AS (scatterd_secret text);
+-- verify the data is retained
+\c - - - :worker_1_port
+SET search_path TO type_conflict;
+\d+ local_table
```

Comment:
Minor: `\d` might change across versions, so can you find the equivalent test from `multi_test_helpers.sql`?